fiftyone/zoo/models/torch.py

```diff
@@ -120,5 +123,5 @@ def custom_load_state_dict_from_url(url, **kwargs):
         entrypoint_module,
         custom_load_state_dict_from_url,
         fcn_name=load_state_dict_from_url.__name__,
-        namespace=torchvision.models,
+        namespace=torchvision,
```

Comment:
Does this change depend on the specific torchvision version? IE will this code still work on `torchvision < 0.12.0`?

Reply:
I'm gonna be honest, I'm not 100% sure how changing the namespace affects this function, I assumed that it just makes it so that all `load_state_dict_from_url` will be replaced with the custom function in the entirety of `torchvision` rather than just in `torchvision.models`. But I did test this with `torchvision < 0.12.0` and it worked.

Reply:
Was this change necessary to get this PR to work?

Reply:
I guess it probably is required or else you wouldn't have done it 😅

Reply:
Ah, you're right. This wasn't necessary. I was under the assumption that since the `load_state_dict_from_url` function definition was moved outside of `torchvision.models` that the namespace would also have to change, but it seems that the namespace only defines where to look for uses of that function and replace those. I'll revert it.